model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/idp/InfinispanIdentityProviderStorageProvider.java

```diff
@@ -207,7 +208,7 @@ public Stream<IdentityProviderModel> getByOrganization(String orgId, Integer fir
             identityProviders.add(idp);
         }
 
-        return identityProviders.stream();
+        return identityProviders.stream().sorted(Comparator.comparing(IdentityProviderModel::getAlias));
```

Comment:
The JPA provider already sorts by alias but the ids are cached as a `HashSet`. Would make sense to use a `LinkedHashSet` instead to keep order and avoid sorting when processing the stream?

Reply:
I don't think this will make much of a difference, but I can change it

Reply:
It doesn't hurt to do it.. I've updated the code to collect the result to be cashed into a LinkedHashSet.